spec/lib/dfid-transition/transform/unescape_html_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+require 'dfid-transition/transform/unescape_html'
+
+module DfidTransition::Transform
+  describe UnescapeHtml do
+    subject(:unescape_html) do
```

Comment:
`unescape_html` sounds like an imperative, this needs a noun – `unescaped_html`?